libnavigation-ui/src/main/java/com/mapbox/navigation/ui/feedback/FeedbackItem.java

```diff
@@ -1,17 +1,19 @@
 package com.mapbox.navigation.ui.feedback;
 
+import com.mapbox.navigation.core.telemetry.events.TelemetryUserFeedback;
+
 public class FeedbackItem {
 
-  // @FeedbackEvent.FeedbackType // TODO Telemetry Impl
+  @TelemetryUserFeedback.FeedbackType
   private String feedbackType;
   private String feedbackText;
   private String description;
   private int feedbackImage;
 
   FeedbackItem(String feedbackText,
                int feedbackImage,
-               // @FeedbackEvent.FeedbackType // TODO Telemetry Impl
-               String feedbackType,
+               @TelemetryUserFeedback.FeedbackType
+                 String feedbackType,
```

Comment:
NIT Unnecessary

Reply:
I don't know where those lines came from but I'll remove them.

Reply:
I meant the formatting issue 😅 Could it be in the same line? `@TelemetryUserFeedback.FeedbackType String feedbackType,`

Anyways, not blocking 👍